### DISCUSSION TITLE
Show question count break down

### DISCUSSION THREAD

**Discussion by kingpinzs** [freeCodeCamp/Developer_Quiz_Site#254] [Ideas]
After playing the RPG game I thought it would be a good idea to add a break down modal to the quiz site that show the number of answers the person got correct.

**Comment by jdwilkin4** [comment]
Are you referring to the modal that appears at the end with the percentage of correct questions for each category?

**Reply by kingpinzs** [reply]
Yeah, I feel that it would help gamify it more and maybe provide more motivation. And maybe in some positive way show what area needs more work.

**Reply by jdwilkin4** [reply]
We can use the Modal component that already comes with react-bootstrap.
We can place this modal in the results page and maybe add a button to open that modal for users to see the percentages.

You can go ahead and create an issue with details on what this feature is about 👍
